flowetl/flowetl/flowetl/operators/staging/sql/event_type_enums.csv

```diff
@@ -0,0 +1,8 @@
+event_type_string, enum
```

Comment:
`interactions.d_event_type` may cover this?

Reply:
It doesn't quite, as things stand at the moment, because we want to identify incoming and outgoing calls/sms as different event types. But I think it would make sense to be able to use `interactions.d_event_type` here.